Mathlib/AlgebraicGeometry/PrimeSpectrum/Basic.lean

```diff
@@ -549,13 +549,13 @@ theorem isIrreducible_iff_vanishingIdeal_isPrime {s : Set (PrimeSpectrum R)} :
     isIrreducible_zeroLocus_iff_of_radical _ (isRadical_vanishingIdeal s)]
 #align prime_spectrum.is_irreducible_iff_vanishing_ideal_is_prime PrimeSpectrum.isIrreducible_iff_vanishingIdeal_isPrime
 
-lemma vanishingIdeal_isIrreducible {R} [CommRing R] :
+lemma vanishingIdeal_isIrreducible {R} [CommSemiring R] :
```

Comment:
Is `R` already a `CommSemiring` in this section?

Reply:
Yes, `section CommSemiring` ends on line 707.